### ISSUE TITLE
Submit an ontology to Ontobee

### ISSUE THREAD

**Issue by labda** [OntoZoo/ontobee#65]
Dear Ontobee team! I would like to include DINTO, an ontology for drug-drug interactions, in Ontobee. This ontology is already listed in the OBO Foundry website and in BioPortal. I would be very grateful if you could inform me about the process. 
Maria

**Comment by cmungall** [comment]
OntoBee should pull automatically from the OBO registry

**Comment by e4ong1031** [comment]
@labda Yes, Ontobee is auto-updating OBO ontologies. But there is one problem regarding DINTO. The PURL link, http://purl.obolibrary.org/obo/dinto.owl, actually link to a github repository, https://github.com/labda/DINTO, instead of the actually OWL file. I am not sure how PURL works, so @cmungall can you please help to resolve the PURL issue?

Thanks

**Comment by cmungall** [comment]
You can add an entry here:
https://github.com/OBOFoundry/purl.obolibrary.org/tree/master/config

click New File

you can use an existing one as template

remember to make a pull request!

**Comment by labda** [comment]
Hi!

Yes, the purl links does not link to the owl file. We'll try to solve that,
but I think we won't be able to do it until next year. So I'll let you know
if, once this is solved, we still have any problem.

Thank you very much,
Maria

**Comment by alanruttenberg** [comment]
When I just looked it seemed that there was only a zip file of the
ontology. Make sure you link to an uncompressed versions. The web client
will land up compressing it for transit anyways.

Alan
